core/local/src/main/proto/user_preferences.proto

```
@@ -0,0 +1,10 @@
+syntax = "proto3";
```

Buf (1.55.1)

Files must have a package defined.

(PACKAGE_DEFINED)